bedrock/base/templates/macros.html

```diff
@@ -349,7 +349,9 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
               <input id="id-input" name="phone-or-email" type="text" required>
             </div>
             <div class="form-submit">
-              <button type="submit" class="button-flat" data-download-os="Android">{{ _('Send') }}</button>
+              <button type="submit" class="button-flat"
+              {% if platform == 'android' %} data-download-os="Android"
+              {% elif platform == 'ios' %} data-download-os="iOS" {% endif %}>{{ _('Send') }}</button>
```

Comment:
I don't see any easy way to pass which option is selected and store that in the data-attribute or I'd update it now. Unless this "select" feature is going live sometime soon, I'd like to just leave it as is.